CHANGELOG.md

```diff
@@ -25,9 +25,15 @@ If possible, provide tooling that performs the changes, e.g. a shell-script.
 
 ## API changes
 
-### The `type_list` header has moved
+### Argument parser
 
-If you included `<seqan3/core/type_list.hpp>` you need to change the path to `<seqan3/core/type_list/type_list.hpp>`.
+* The signature of input file and output file validator have been modified to template classes.
+  It is now possible to get the list of file extensions from the type of a seqan input file, respectively output file. Old code will likely not be affected unless the type was used explicitly.
+
+### Core
+
+* The `type_list` header has moved
+  If you included `<seqan3/core/type_list.hpp>` you need to change the path to `<seqan3/core/type_list/type_list.hpp>`.
```

Comment:
This will be joined into one line with the previous line in our doxygen documentation. I would prefer to keep  the `###` per item so that we have enough space to describe a feature or API workaround.

Reply:
But no one is looking at our changelog through doxygen. I propose highlighting the begin of the point with bold and adding a colon. This will be rendered properly in markdown and doxygen too.